packages/api/tests/models/test-granules-model.js

```diff
@@ -213,3 +220,115 @@ test('files existing at location returns only file that exists with multiple des
     recursivelyDeleteS3Bucket(destBucket2)
   ]);
 });
+
+test('get() will translate an old-style granule file into the new schema', async (t) => {
+  const oldFile = {
+    bucket: 'my-bucket',
+    filename: 's3://my-bucket/path/to/file.txt',
+    filepath: 'path/to/file.txt',
+    name: 'file123.txt',
+    path: 'source/path',
```

Comment:
Do we store the new `source` field? Correct me if I'm wrong, but I thought `path` (as in the provider path) was translated to `source` in the new schema. Is that right? I don't see `source` in the expectation for this test.

Reply:
The `source` property is added to a file if information about the provider is available.  In the case of a granule (and its associated files) being added when a Cumulus message is send through `es/indexer.handler()`, the provider information should be available.  When fetching a granule from the database, though, that provider information is not available, so the `source` can't be generated.